bond/bond.py

```diff
@@ -379,6 +379,10 @@ def get_param_groups_dataframes(self):
         summary = _order_columns(pd.concat(param_group_summaries,
                                  ignore_index=True))
 
+        # sort summary csv by count in descending order
```

Comment:
This is the wrong place for the sort to happen. It should happen before the param group numbers are assigned, otherwise they won't match the keys/params in the files csv

Reply:
Ah good point. I forgot about the files csv! I moved the sort to the function you suggested and pushed. CircleCi just passed. Let me know if there are any more issues!